temp.c

```diff
@@ -173,65 +259,22 @@ void temp_sensor_tick() {
 
 				#ifdef	TEMP_THERMISTOR
 				case TT_THERMISTOR:
-					do {
-						uint8_t j, table_num;
-						//Read current temperature
-						temp = analog_read(i);
```

Comment:
This was actually a bug. Should be `temp = analog_read(temp_sensors[i].temp_pin)` because we want to read ADC pin of i-th sensor rather then ADC pin i. Fixed.